spec/fixtures/files/bitklavier_binaural.json

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
Notice that the new fixtures have a `resource` property but also `collection` and `files`, hence the big refactor.